src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -21,6 +21,10 @@
       <File Include="$(CoreCLRBuildIntegrationDir)*" TargetPath="build" />
       <File Include="$(CoreCLRILCompilerDir)netstandard\*" TargetPath="tools/netstandard" />
       <File Include="sdk\Sdk.props" TargetPath="Sdk" />
+      <File Include="$(MSBuildThisFileDirectory)\..\..\..\..\native\libs\System.Globalization.Native\*" TargetPath="source_code/libs/System.Globalization.Native"/>
```

Comment:
There is actually an existing directory structure convention for native source. Could you please rename `source_code` to `native/src` ?

For example, https://nuget.info/packages/Microsoft.NETCore.App.Runtime.browser-wasm/6.0.0-preview.3.21201.4 package has `native/src` directory like that.